### ISSUE TITLE
Improve standard library signature coverage

### ISSUE THREAD

**Issue by soutaro** [soutaro/ruby-signature#1]
We provide the signatures of a very limited number of standard library classes. We will really appreciate if you write the signature of other classes.

(Note that we are currently working for standard library signatures.)

**Comment by connorshea** [comment]
Sorbet already has type signatures for a lot of the stdlib, have you considered writing something to transform their files into ruby-signature-compatible files to automate this process?

https://github.com/sorbet/sorbet/tree/master/rbi/core

**Comment by soutaro** [comment]
Yes, it should be one of the best ways to improve the coverage. I haven't tried yet.

**Comment by connorshea** [comment]
I know this ended up partially happening in #23, do we have any way of tracking progress for stdlib coverage? Essentially code coverage for type signatures. :)

**Comment by soutaro** [comment]
I have a guide for contributions at https://github.com/ruby/ruby-signature/blob/master/docs/CONTRIBUTING.md . Could you check it? Tell me if you have any questions or feedback.

**Comment by connorshea** [comment]
For a lot of the libraries in the spreadsheet, we should be able to get the signatures from Sorbet e.g. for [`csv`](https://github.com/sorbet/sorbet/blob/master/rbi/stdlib/csv.rbi) or [`benchmark`](https://github.com/sorbet/sorbet/blob/master/rbi/stdlib/benchmark.rbi).

**Comment by soutaro** [comment]
Yes, it should be a good option to start having an initial version. However, I believe we need to review and revise them ourselves anyway.

**Comment by gaurav2699** [comment]
Hey, is this issue still open? I would like to work on this issue.

**Comment by Harshit2929** [comment]
Is this issue still open?

**Comment by snaka** [comment]
Hi.
It seems that @gaurav2699 's question hasn't been answered until now, so I'd like to make sure.

Is it correct that this issue is still active?

Also, have you already implemented the idea of importing the sorbet type definition file?

I'm hesitant to start a contribution because I personally can't read the status of this issue.

Thank you for your time.

**Comment by snaka** [comment]
I' ve read the code and the PR, and it seems to be still open, so I'll try to help as much as I can.
Thank you very much.

> Also, have you already implemented the idea of importing the sorbet type definition file?

It seems to have been addressed in the following PR 👍 

https://github.com/ruby/rbs/pull/24

**Comment by snaka** [comment]
I thought it would be better to change the title of this issue to "core and standard library".

Reference : https://github.com/ruby/rbs/issues/876
